From 43d8efbdfc9c3e8bde7e4df7b406692ef4ecaadb Mon Sep 17 00:00:00 2001
From: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Date: Mon, 3 Aug 2015 16:29:51 +0900
Subject: [PATCH 53/85] drm: rcar-du: Add GEN3 and R8A7793 dependence

This patch adds the device of R-Car Gen3 and R8A7793 to depends on.
In addition, this patch adds the device of R8A7793 and R8A7795 to help.

Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
---
 drivers/gpu/drm/rcar-du/Kconfig |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/rcar-du/Kconfig b/drivers/gpu/drm/rcar-du/Kconfig
index 04e5335a..174214d 100644
--- a/drivers/gpu/drm/rcar-du/Kconfig
+++ b/drivers/gpu/drm/rcar-du/Kconfig
@@ -22,10 +22,11 @@ config DRM_RCAR_HDMI
 config DRM_RCAR_LVDS
 	bool "R-Car DU LVDS Encoder Support"
 	depends on DRM_RCAR_DU
-	depends on ARCH_R8A7790 || ARCH_R8A7791 || COMPILE_TEST
+	depends on ARCH_R8A7790 || ARCH_R8A7791 || ARCH_R8A7793 || \
+			ARCH_RCAR_GEN3 || COMPILE_TEST
 	help
 	  Enable support for the R-Car Display Unit embedded LVDS encoders
-	  (currently only on R8A7790 and R8A7791).
+	  (currently only on R8A7790, R8A7791, R8A7793 and R8A7795).
 
 config DRM_RCAR_VSP
 	bool "R-Car DU VSP Compositor Support"
-- 
1.7.9.5

